betelgeuse.py

```diff
@@ -574,6 +621,40 @@ def cli(context, jobs):
     testimony.SETTINGS['minimum_tokens'] = ['id']
 
 
+@cli.command('create-plan')
+@click.option(
+    '--name',
+    default='test-plan-{0}'.format(time.time()),
+    help='Name for new Test Plan.',
+)
+@click.option(
+    '--type',
```

Comment:
I would name this option plan-type to not override the `type` builtin on the body of the command function.